javascripts/app.js

```diff
@@ -30,6 +29,62 @@ console.log("spell: ", spell.toString());
    */
   $("#player-setup").show();
 
+  // Display Name
+  $("#nameChoice").on("click", function() {
+    warrior.name = $("#player-name").val();
+    $("#battleName").append(warrior.name);
+  });
+
+  $(".classChoice").on("click", function() {
+    var humanClass = this.id
+
+    switch(humanClass) {
```

Comment:
You're right, that is way cleaner. Just using an if statement. Thanks for
the challenge.

On Sat, Apr 23, 2016 at 8:14 PM, Steve Brownlee <notifications@github.com>
wrote:

> In javascripts/app.js
> <https://github.com/nss-evening-cohort-3/gauntlet-destroy-in-ators/pull/12#discussion_r60837413>
> :
>
> > @@ -30,6 +29,62 @@ console.log("spell: ", spell.toString());
> >     */
> >    $("#player-setup").show();
> >
> > +  // Display Name
> > +  $("#nameChoice").on("click", function() {
> > +    warrior.name = $("#player-name").val();
> > +    $("#battleName").append(warrior.name);
> > +  });
> > +
> > +  $(".classChoice").on("click", function() {
> > +    var humanClass = this.id
> > +
> > +    switch(humanClass) {
>
> This switch statement is completely unnecessary. It can be replaced with a
> 4 line if statement.
>
> if (this.id === "surprise") {
>   // do something
> } else {
>   // do something else
> };
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/nss-evening-cohort-3/gauntlet-destroy-in-ators/pull/12/files/46da2d22a284a2f98e51be774918ac43ab104b62#r60837413>
>



-- 
Jimmy T. Perry BS, HFS
615-585-1433
jtmp2r@gmail.com